mousebuttons.go

```diff
@@ -26,4 +26,10 @@ const (
 	MouseButtonLeft   MouseButton = ui.MouseButtonLeft
 	MouseButtonRight  MouseButton = ui.MouseButtonRight
 	MouseButtonMiddle MouseButton = ui.MouseButtonMiddle
+	MouseButton4      MouseButton = ui.MouseButton4
```

Comment:
Start with 3.

Reply:
By the way, do we need so many buttons? IMO two extra mouse buttons are enough.

Windows has only 2 extra buttons:
https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-mouse_event
https://github.com/glfw/glfw/blob/d299d9f78857e921b66bdab42c7ea27fe2e31810/src/win32_window.c#L810-L819

Unclear about macOS:
https://github.com/glfw/glfw/blob/d299d9f78857e921b66bdab42c7ea27fe2e31810/src/cocoa_window.m#L483

Unclear about Linux (X11) but probably there are only 2 extra button:
https://github.com/glfw/glfw/blob/d299d9f78857e921b66bdab42c7ea27fe2e31810/src/x11_window.c#L1380-L1385
https://tronche.com/gui/x/xlib/events/processing-overview.html